.gitignore

```diff
@@ -1,4 +1,5 @@
 # General ignores.
+cscope.out
```

Comment:
What is this used for?

Reply:
It's kind of like ctags.

Reply:
Is it intrinsic to your change?  Should it be in a different PR?

Reply:
It's intrinsic to my comfort :(

Reply:
I'm not arguing that it shouldn't be committed :) Just that if something causes us to merge and then revert this PR, it would be a shame to have to recommit and merge this modification to .gitignore

Reply:
I went ahead and removed it.